packages/htmlbars-compiler/lib/hydration-opcode-compiler.js

```diff
@@ -233,8 +222,13 @@ HydrationOpcodeCompiler.prototype.pushMorphPlaceholderNode = function() {
   this.comment();
 };
 
+HydrationOpcodeCompiler.prototype.MustacheStatement = function(mustache) {
```

Comment:
Why is this here? Seems strange to have both this AND `HydrationOpcodeCompiler.prototype.mustache`, no?